template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ConfigUtils.java

```diff
@@ -66,7 +67,7 @@ public static Optional<StorageLocationView> getStorageLocationForServiceProperty
     }
 
     public static String getSafetyValveProperty(String key, String value) {
-        return String.format(CM_SAFETY_VALVE_PROPERTY_FORMAT, key, value);
+        return String.format(CM_SAFETY_VALVE_PROPERTY_FORMAT, key, StringEscapeUtils.escapeXml11(value));
```

Comment:
I fear this will cause double-escaping of reserved characters in `RangerRoleConfigProvider.addDbSslConfigsIfNeeded()`. (Namely, `&` in XML entities will be mapped to `&amp;`, so e.g. `&lt;` in the input will likely become `&amp;lt;`.) :worried:

Reply:
Nevermind, I just noticed your update in `RangerRoleConfigProvider`.